webforj-plugins/webforj-install-bbj-plugin/src/test/resources/log4j2.xml

```diff
@@ -6,6 +6,6 @@
     </Console>
   </Appenders>
   <Loggers>
-    <Root level="debug"/>
+    <Root level="warn"/>
```

Comment:
This is a log4j2.xml configuration for the unit test only.  Since I test errors, things that cause exceptions, the console or the terminal would fill up with content that I did not need to see.